airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -154,7 +158,7 @@ def get_async_session(self):
 
         return async_get_session()
 
-    def create_session(self, deferrable: bool = False) -> boto3.session.Session:
+    def create_session(self, deferrable: bool = DEFAULT_DEFERRABLE) -> boto3.session.Session:
```

Comment:
I made a comment on this earlier, but the `deferrable` parameter should be `False` here, not `DEFAULT_DEFERRABLE`. The reason is that although a user may want to use operators in deferrable mode, the operators still need to be able to get a regular (synchronous) boto3 client. When an operator uses deferrable mode, it accesses the `async_conn` property of the hook it is using, and the `async_conn` property is what sets `deferrable=True` and gets the async client.

Reply:
Sorry for missing this one. It got reverted during the changes 🤦‍♂️ Just updated it!